src/telemetry/walkthroughStateProvider.ts

```diff
@@ -87,7 +87,16 @@ const walkthroughRequiredMapping: Readonly<Map<WalkthroughContextKeys, Walkthrou
 		{
 			subscriptionStates: triedProStates,
 			subscriptionCommands: tryProCommands,
-			usage: [`command:${Commands.CreateCloudPatch}:executed`, `command:${Commands.CreatePatch}:executed`],
+			usage: [
+				`patchDetailsView:shown`,
+				`patchDetailsWebview:shown`,
+				`draftsView:shown`,
+				`command:${Commands.ShowDraftsView}:executed`,
+				`command:${Commands.ShowPatchDetailsPage}:executed`,
+				`command:${Commands.CreateCloudPatch}:executed`,
+				`command:${Commands.CreatePatch}:executed`,
+				`command:${Commands.FocusDraftsView}:executed`,
```

Comment:
this command is no longer used in the walkthrough

Reply:
removed